abstract_lungair_data.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
+
+class AbstractLungairData(ABC):
+	"""Abstract class for loading patient data into a smart FHIR server"""
+	def __init__(self):
+		pass
+
+	def getAllPatients(self):
+		"""Returns a list of all patients."""
+		pass
```

Comment:
It's possible to decorate these methods with `@abstractmethod` to force non-abstract derived classes to provide an implementation; see https://docs.python.org/3/library/abc.html#abc.abstractmethod